src/CONFIG/pyproject.toml

```diff
@@ -0,0 +1,24 @@
+[tool.poetry]
+name = "qat-config"
```

Comment:
Likely want it to be `qat-compiler-config` to make sure there's no confusion with other projects.